jumpscale/sals/vdc/threebot.py

```diff
@@ -32,9 +32,10 @@ def deploy_threebot(self, minio_wid, pool_id, kube_config):
             "VDC_OWNER_TNAME": self.vdc_deployer.tname,
             "VDC_EMAIL": self.vdc_deployer.email,
             "VDC_PASSWORD_HASH": self.vdc_deployer.password_hash,
-            "VDC_WALLET_SECRET": self.vdc_deployer.wallet.secret,
-            "VDC_INSTANCE": j.data.serializers.json.dumps(vdc_dict),
             "KUBE_CONFIG": kube_config,
+            "PROVISIONING_WALLET_SECRET": self.vdc_deployer.vdc_instance.provision_wallet.secret,
+            "PREPAID_WALLET_SECRET": self.vdc_deployer.vdc_instance.prepaid_wallet.secret,
+            "VDC_INSTANCE": j.data.serializers.json.dumps(self.vdc_instance.to_dict()),
```

Comment:
why using `.to_dict()` again? this will send a very large env variable that includes info about `k8s` and possibly `minio` that are not required
it used to pass a minimal dict which is created above `vdc_dict`
```suggestion
            "VDC_INSTANCE": j.data.serializers.json.dumps(vdc_dict),
```